client/css/widgets/pile.css

```diff
@@ -34,3 +38,7 @@
 body.edit .pile {
   display: none;
 }
+
+.pile > :last-child, .pile > :nth-last-child(2) {
```

Comment:
This may help sometimes, but since DOM order is not always z order, the top cards will not always get this optimization. I'm curious how much this helps without this part of the code.

Reply:
Tests on production adding `will-change: transform`  in the developer tools window directly to the pile element seems to help. I recommend keeping `will-change: transform` for the pile only.

Reply:
You are right, sort and shuffle will change the z order without changing the DOM order.  That will only help when cards are placed on a pile (which is what I was doing while testing).  `Putting will-change: transform` on the pile made the biggest difference.  However, cards in the default deck came off the deck a bit slow, and adding `will-change: transform` to the top two cards appeared to help. 

That needs to be done by z-value though, so I'll go ahead and remove this part.  With the new deck image the difference with and without it on the top two cards seems minimal.  If I figure out a way to apply it by z-order later, then perhaps that can be tested to see if it is worthwhile.